components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/java/org/wso2/carbon/apimgt/rest/api/store/mappings/APIMappingUtil.java

```diff
@@ -82,8 +82,8 @@ public static APIDTO toAPIDTO(API api) {
         apiDTO.setVersion(api.getVersion());
         apiDTO.setContext(api.getContext());
         apiDTO.setDescription(api.getDescription());
-        apiDTO.setPolicies(api.getPolicies());
-        apiDTO.setLabels(api.getLabels());
+        apiDTO.setPolicies(new ArrayList<>(api.getPolicies()));
```

Comment:
api.getPolicies() now returns a Set. Usage of Sets is desirable to indicate that a collection should not have duplicates(which is the case with policies and labels). But the auto generated apiDTO.setPolicies() accepts a List.

So the only way to convert between Set to List is to construct a new List instance with the respective Set as an argument.